getting-started-guides/csp/gcp/gcp.md

```diff
@@ -200,7 +200,7 @@ Submitting a CPU job on this cluster is very similar. Below's an example command
 ```bash
     export GCS_BUCKET=my-bucket
     export MAIN_CLASS=ai.rapids.spark.examples.mortgage.CPUMain
-    export RAPIDS_JARS=gs://$GCS_BUCKET/spark-gpu/sample_xgboost_apps-0.1.4-jar-with-dependencies.jar
+    export RAPIDS_JARS=gs://$GCS_BUCKET/spark-gpu/sample_xgboost_apps-0.1.5-jar-with-dependencies.jar
     export DATA_PATH=gs://$GCS_BUCKET
     export TREE_METHOD=hist
     export SPARK_NUM_EXECUTORS=4
```

Comment:
Do we really need gcp/pics/gcp-bucket.png?

Reply:
rm file : pics/gcp-bucket.png